src/modules/sector-industries/dto/get-sector-industries-query.dto.ts

```diff
@@ -6,13 +6,13 @@ export class GetSectorIndustriesQueryDto {
   @IsString()
   @MaxLength(4)
```

Comment:
Can we also add `@Min(2)`, or a number you think is a common min standard please?